index.bs

```diff
@@ -2255,15 +2249,9 @@ When this method is invoked, the user agent MUST execute the following algorithm
         is not present, then set |lifetimeTimer| to a [=client=]-specific default.
 
             Recommended ranges and defaults for <code>|pkOptions|.{{PublicKeyCredentialRequestOptions/timeout}}</code> are as follows.
-              If <code>|pkOptions|.{{PublicKeyCredentialRequestOptions/userVerification}}</code>
                 <dl class="switch">
-                    :   is set to {{UserVerificationRequirement/discouraged}}
-                    ::  Recommended range: 30000 milliseconds to 180000 milliseconds.
-                    ::  Recommended default value: 120000 milliseconds (2 minutes).
-
-                    :   is set to {{UserVerificationRequirement/required}} or {{UserVerificationRequirement/preferred}}
-                    ::  Recommended range: 30000 milliseconds to 600000 milliseconds.
-                    ::  Recommended default value: 300000 milliseconds (5 minutes).
+                    :  Recommended range: 300000 milliseconds to 600000 milliseconds.
+                    :  Recommended default value: 300000 milliseconds (5 minutes).
                 </dl>
```

Comment:
The `switch` construct doesn't seem quite appropriate for this. I suggest replacing the `<dl>`s with the bulleted list syntax like at https://github.com/w3c/webauthn/pull/1885/files#diff-5e793325cd2bfc452e268a4aa2f02b4024dd9584bd1db3c2595f61f1ecf7b985R1801-R1803 .